web/docs/about.mdx

```diff
@@ -0,0 +1,131 @@
+---
+id: about
+title: Introduction
+description: 'What is Supabase?'
+slug: /
+hide_table_of_contents: true
+---
+
+import Link from '@docusaurus/Link'
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+const frameworks = [
+  { name: 'Angular', logo: '/img/libraries/dart-icon.svg', href: '/docs/guides/with-angular' },
+  {
+    name: 'Expo',
+    logo: '/img/libraries/expo-icon.svg',
+    href: 'https://github.com/supabase/supabase/tree/master/examples/expo-todo-list',
+  },
+  { name: 'Flutter', logo: '/img/libraries/dart-icon.svg', href: '/docs/guides/with-flutter' },
+  {
+    name: 'Javascript',
+    logo: '/img/libraries/javascript-icon.svg',
+    href: 'https://github.com/supabase/supabase/tree/master/examples/javascript-auth',
+  },
+  { name: 'Next.js', logo: '/img/libraries/nextjs-icon.svg', href: '/docs/guides/with-nextjs' },
+  { name: 'React', logo: '/img/libraries/react-icon.svg', href: '/docs/guides/with-react' },
+  { name: 'Svelte', logo: '/img/libraries/svelte-icon.svg', href: '/docs/guides/with-svelte' },
+  { name: 'Vue', logo: '/img/libraries/vuejs-icon.svg', href: '/docs/guides/with-vue-3' },
+]
```

Comment:
Missing icons for angular, and flutter (although the Dart icon is probably acceptable here)